gobblin-kubernetes/gobblin-service/base-cluster/deployment.yaml

```diff
@@ -22,18 +22,17 @@ spec:
         - name: 'shared-jobs'
           persistentVolumeClaim:
             claimName: shared-jobs-claim
-        - name: 'shared-template-catalogs'
-          persistentVolumeClaim:
-            claimName: shared-template-catalogs-claim
+        - name: flowconfig-templates
```

Comment:
Are we introducing `shared-template-catalog` persistentVolumeClaim with configMap?

Reply:
I'm going to deprecate `shared-template-catalog` because the `flowconfig-templates` volume is applied for every pod, and it loads the templates from the `base` folder